esbuild.mjs

```
@@ -0,0 +1,22 @@
+import esbuild from "esbuild";
+
+let outfile = "lib/bundle.js";
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'outfile' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)